lib/api/index.js

```diff
@@ -1,4 +1,18 @@
+'use strict'
+
+/**
+ * Handles a request to a not implemented `/api/` endpoint.
+ * @param req
+ * @param res
+ * @param next
+ */
+function notImplemented (req, res, next) {
+  res.status(404).send('API not implemented')
```

Comment:
Like I said I can understand it if the content type is application/json but otherwise it seems redundant to me.